node/tests/SharedTests.ts

```diff
@@ -12287,6 +12287,20 @@ export function runBaseTests(config: {
                         await client.set("test_key2", "test_value2");
                         const value2 = await client.get("test_key2");
                         expect(value2).toEqual("test_value2");
+                        // Testing that after kill update password works with `reAuth: true` works
+                        await client.customCommand([
```

Comment:
I move it, as in python to seperate test suite. The AUTH is a whole module of different functionalities and im creating the pltform for it with the feature, making it ready for next tasks.
As part of another suite it will close the the cluster after finishing.
You should do the same in your PR.